xml/MAIN-SBP-CSP-UpdateInfra.xml

```diff
@@ -413,6 +422,107 @@ ssh-keygen -t rsa -f regionsrv</screen>
 					</step>
 				</procedure>
 			</sect3>
+			<sect3 xml:id="sec-configure-data-disks">
+				<title>Configuring Data Disks to Hold Repositories and DB</title>
+				<para> After the installation of the RMT server package, the directory structure
+					required by RMT is set up in <filename>/var/lib/rmt/public/repo</filename>. The
+					repositories should <emphasis role="strong">not</emphasis> be stored on the root
+					volume. Storing the repository cash on the root volume will significantly
+					increase the recovery time should the RMT system experience issues and need to
+					be re-created. The following process outlines the steps necessary to set up the
+					external device to hold the repositories. </para>
+				<para> For the RMT repository disk follow the steps outlined below: </para>
+				<procedure>
+					<step>
+						<para> Add the disk to the SUSE Linux Enterprise Server instance that holds
+							the RMT server. </para>
+					</step>
+					<step>
+						<para> Use <command>sudo fdisk -l</command> to list the available block
+							devices. </para>
```

Comment:
It seems to me that this assumes MBR disks, i.e. volumes under 2TB. This is **not** a safe assumption any longer in 2021.

Reply:
Changed to "sudo lsblk".